packages/terra-notification-dialog/src/terra-dev-site/doc/example/ConfirmNotificationDialogExample.jsx

```diff
@@ -0,0 +1,65 @@
+import React from 'react';
+import { injectIntl, intlShape } from 'terra-base';
```

Comment:
We need to import `injectIntl, intlShape` this from `react-intl`. Pulling from terra-base ends up executing more of terra-base than just the `injectIntl` and `intlShape`. 
See https://github.com/cerner/terra-core/issues/1982 for more info.